src/helm/benchmark/model_deployment_registry.py

```diff
@@ -6,32 +7,52 @@
 
 from helm.common.hierarchical_logger import hlog
 from helm.common.object_spec import ObjectSpec
+from helm.proxy.models import ALL_MODELS, FULL_FUNCTIONALITY_TEXT_MODEL_TAG, MODEL_NAME_TO_MODEL, TEXT_MODEL_TAG, Model
+
+
+MODEL_DEPLOYMENTS_FILE = "model_deployments.yaml"
 
 
 class ClientSpec(ObjectSpec):
     pass
 
 
+class WindowServiceSpec(ObjectSpec):
+    pass
+
+
 @dataclass(frozen=True)
 class ModelDeployment:
     """A model deployment is an accessible instance of this model (e.g. a hosted endpoint).
 
-    A model can have model deployments."""
+    A model can have multiple model deployments."""
 
     name: str
     """Name of the model deployment."""
 
-    model_name: str
-    """Name of the model that this model deployment is for."""
-
     client_spec: ClientSpec
     """Specification for instantiating the client for this model deployment."""
 
-    max_sequence_length: Optional[int]
-    """Maximum equence length for this model deployment."""
+    model_name: Optional[str] = None
```

Comment:
Why is this moved down?

Reply:
The ordering should be that all the required parameters come first, then all the optional parameters with default arguments.